src/express/common/remote-worker.js

```diff
@@ -47,6 +47,42 @@ export async function runSshCommand(ip, command, retries) {
   }
 }
 
+export async function runSshCommandWithReturn(ip, command, retries) {
```

Comment:
Why is this needed?

Reply:
We needed to have the stdout in a variable which is not possible by the legacy `runSshCommand`, hence this was introduced.

Reply:
Yeah sure, I meant more like why we need the stdout var to be returned in the first place.
Also, maybe this can be merged in the previous function and avoid duplications by using some sort of extracted util function

Reply:
This function is used for returning the value of `eth.getBlock(num)` which was not possible with the `runSshCommand`.